lib/rules/no-var.js

```diff
@@ -346,9 +346,19 @@ module.exports = {
 
 		return {
 			"VariableDeclaration:exit"(node) {
-				if (node.kind === "var") {
```

Comment:
Currently, this rule doesn't support TS syntax. If we add TS support to this rule, we need to add the following meta properties in the schema.

https://github.com/eslint/eslint/blob/58a171e8f0dcc1e599ac22bf8c386abacdbee424/lib/rules/no-loss-of-precision.js#L16-L17